### ISSUE TITLE
Add alt text to images

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/techdocs#113]
Images on the Accord Project tech docs lack alternative text to describe the information for people with visual impairments.
https://docs.accordproject.org/

Here are some helpful resources:
https://www.w3.org/WAI/tutorials/images/
https://moz.com/learn/seo/alt-text

If you need help deciding what to make the alt text for an image, let me know.

Example of image with no alt text:
![image](https://user-images.githubusercontent.com/40569793/66154580-cb284500-e5eb-11e9-8e53-6f1f5c874c91.png)

**Comment by Alexandria** [comment]
Looks awesome! I can take a look at this.

**Comment by Michael-Grover** [comment]
@Alexandria that would be great, thanks!

**Comment by Alexandria** [comment]
@Michael-Grover I see a lot of these pics have a generic alt message like "Advance-Late-1". Are you wanting those updated/changed or are you only concerned with the images that have no alt text at all?  Thanks!

**Comment by Michael-Grover** [comment]
@Alexandria more informative alt text would be good. The resources I sent in the issue are helpful for writing good alt text. Thanks for pointing this out and asking!

**Comment by Alexandria** [comment]
> @Alexandria more informative alt text would be good. The resources I sent in the issue are helpful for writing good alt text. Thanks for pointing this out and asking!

Yup, I've been looking through those links. They have been supper helpful so far.

**Comment by djiwandou** [comment]
Hi @Michael-Grover could I also try to solve this issue, I am still newbie though 😅 ✌️ 
thanks @Alexandria for made it clearer 👍

**Comment by Michael-Grover** [comment]
@djiwandou, I'll leave that up to @Alexandria since they've claimed this issue already. Thank you for being interested in helping!

**Comment by Alexandria** [comment]
@djiwandou Go for it! I am currently working on the home page(I'll open a draft request later). I did a quick run through the application and there is certainly enough work for two peeps. I'm also new too so no worries 👍

@Michael-Grover I had a question about the code base. Like I mentioned above I am working on the homepage. I see in the code where the image is being added. However I am not sure how to add the alt property to it. 

I am looking here `/website/pages/index.js` 
<img width="1220" alt="Screen Shot 2019-10-04 at 9 47 37 AM" src="https://user-images.githubusercontent.com/7613670/66217065-36c1ef00-e68c-11e9-8265-ea8761210611.png">

This seems to correlate to this image on the homepage.
<img width="961" alt="Screen Shot 2019-10-04 at 9 50 06 AM" src="https://user-images.githubusercontent.com/7613670/66217137-5f49e900-e68c-11e9-9362-8ab88191af0a.png">

It looks like the `imgUrl()` function may be creating the `<img>` tag but I am not sure. Would you be able to explain how I would go about adding the alt property in this case or point me in the right direction?

**Comment by Michael-Grover** [comment]
@Alexandria , I think @jeromesimeon would be able to answer your question. I'm not familiar with how this subsite is coded.

**Comment by irmerk** [comment]
@Alexandria these docs are made from [Docusaurus](https://docusaurus.io/), I would look into their developer docs which could be helpful.

**Comment by Snaked06** [comment]
Hey @Alexandria @Michael-Grover @jeromesimeon , i love html can i take this issue would love to contribute

**Comment by aditya1rawat** [comment]
Is this still available to do? I am pretty good at HTML and would love to take this on!

**Comment by irmerk** [comment]
> Hey @Alexandria @Michael-Grover @jeromesimeon , i love html can i take this issue would love to contribute

> Is this still available to do? I am pretty good at HTML and would love to take this on!

@Snaked06 and @aditya1rawat, up to @Alexandria and @djiwandou if they need help with the issue.

**Comment by Michael-Grover** [comment]
@Alexandria your PR looks great, I will add my review soon. Was anyone able to help you add alt text to the interior pages?

https://deploy-preview-135--accordproject-old.netlify.com/docs/accordproject.html

**Comment by Alexandria** [comment]
@Snaked06  and @aditya1rawat no additional help is needed right now with this one, but thanks!

**Comment by Alexandria** [comment]
@Michael-Grover I figured out how to modify the alt text in the internal webpages. I'll try to have another pull request buy the end of the day 👍

**Comment by Michael-Grover** [comment]
@Alexandria , awesome, thank you!

**Comment by Michael-Grover** [comment]
if @Alexandria is done with this, we can close it and open another one for improving the alt text of interior pages.

**Comment by irmerk** [comment]
Once that issue is created we can close this issue.

**Comment by devcer** [comment]
I found a couple of images which doesn't have an alt text yet. added the related labels and submitted a PR.

**Comment by devcer** [comment]
I see there are still a lot of other A11Y issues in this repo. Would you like me to go ahead and fix them?
<img width="1333" alt="Screenshot 2019-10-18 at 10 56 18 AM" src="https://user-images.githubusercontent.com/3799600/67068845-ddcc7d80-f198-11e9-8ff7-0041e3871a1e.png">

**Comment by irmerk** [comment]
@devcer yes, I think making incremental issues for A11Y related work, then working off of them would be great.

**Comment by snehit31** [comment]
I would like to work on this issue.

**Comment by jeromesimeon** [comment]
@Michael-Grover @irmerk Is this done?

**Comment by irmerk** [comment]
Closing for now, as mentioned https://github.com/accordproject/techdocs/pull/221#issuecomment-551929461 @Alexandria can reopen with a new PR.
